### PR TITLE
update area and xpanse Api

### PR DESCRIPTION
**Pull request by WangLiNaruto**

### CODE REVIEW

File: src/xpanse-api/generated/apis/AdminApi.ts [hunk 1]
```diff
@@ -1,21 +1,21 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- * SPDX-FileCopyrightText: Huawei Inc.
- */
+// TODO: better import syntax?
```
Comment (iskey) [inline src/xpanse-api/generated/apis/AdminApi.ts:1]: please remove this TODO, keep the license here.

File: src/components/utils/Area.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/**
```
Comment (iskey) [inline src/components/utils/Area.ts:1]: please add the license here.

File: src/xpanse-api/generated/configuration.ts [hunk 3]
```diff
@@ -1,13 +1,8 @@
...
- * SPDX-License-Identifier: Apache-2.0
- * SPDX-FileCopyrightText: Huawei Inc.
- */
-
-import { HttpLibrary } from './http/http';
-import { Middleware, PromiseMiddleware, PromiseMiddlewareWrapper } from './middleware';
-import { IsomorphicFetchHttpLibrary as DefaultHttpLibrary } from './http/isomorphic-fetch';
-import { BaseServerConfiguration, server1 } from './servers';
-import { AuthMethods, AuthMethodsConfiguration, configureAuthMethods } from './auth/auth';
+import { HttpLibrary } from "./http/http";
```
Comment (iskey) [inline src/xpanse-api/generated/configuration.ts:1]: Please revert the SPDX license back.

**Comment by iskey** [comment]
Did you forget to reformat the codes? 
`npx prettier --config .prettierrc --write .` will be useful.

**Review by swaroopar — CHANGES_REQUESTED** [review]
please format code. After that hopefully we will be able to see the actual code differences with this PR.

File: src/xpanse-api/generated/models/Region.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+ * Do not edit the class manually.
+ */
+
+import { HttpFile } from '../http/http';
+
+/**
+ * The regions of the Cloud Service Provider
+ */
+export class Region {
+    'name'?: string;
```
Comment (swaroopar) [inline src/xpanse-api/generated/models/Region.ts:24]: These fields should not be optional.
please generate the client after adding the OpenAPI validations which i mentioned in the Xpanse PR. 
After that.. these two properties in Region class will not be marked as optional.

Comment (iskey) [reply src/xpanse-api/generated/models/Region.ts:24]: the field `name` is mandatory, but the `area` should be optional, I think.

**Review by iskey — APPROVED** [review]
looks good.

**Review by swaroopar — APPROVED** [review]
looks good.

**Pull request merged**